appveyor.yml

```diff
@@ -62,11 +62,11 @@ install:
 
   # Install the dependencies of hyperspy.
   - ps: Add-AppveyorMessage "Installing conda packages..."
-  - if "%PLATFORM%"=="x64" set DEPS=%DEPS% mrcz tifffile
+  - if "%PLATFORM%"=="x64" set DEPS=%DEPS% mrcz tifffile==2020.2.16
   - "conda install -y %DEPS% %TEST_DEPS%"
```

Comment:
As above - on another matter - in pyxem we've removed appveyor completely by using choco to test on Windows using Travis, perhaps that would be good for hyperspy too?

Reply:
Indeed, appveyor runs only one build at a time, which is quite slow... and travis also offers limited capacity... I have setup a CI repo last year to start using azure pipeline with the idea that these scripts would be easily reusable and we can used them from various repos. It works fairly well https://github.com/ericpre/CI, I have been using it to build the bundle https://github.com/ericpre/hyperspy-bundle/tree/test_release_cross_platform.